package.json

```diff
@@ -19,13 +19,22 @@
   "license": "ISC",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/blackbaud/skyux-sample.git"
+    "url": "git+https://github.com/blackbaud/skyux-link-records.git"
   },
   "bugs": {
-    "url": "https://github.com/blackbaud/skyux-sample/issues"
+    "url": "https://github.com/blackbaud/skyux-link-records/issues"
+  },
+  "homepage": "https://github.com/blackbaud/skyux-link-records#readme",
+  "peerDependencies": {
+    "@angular/common": "^4.3.6",
+    "@angular/core": "^4.3.6",
+    "@angular/router": "^4.3.6",
+    "@skyux/core": "^3.1.0",
+    "@skyux/i18n": "^3.0.1",
+    "@skyux/indicators": ">=3.0.0-rc.0",
+    "@skyux/forms": "^3.0.0-rc.0",
+    "rxjs": "^5.4.3"
   },
-  "homepage": "https://github.com/blackbaud/skyux-sample#readme",
-  "peerDependencies": {},
   "dependencies": {},
```

Comment:
Need to add `microedge-rxstate` to dependencies.

Reply:
added